### ISSUE TITLE
Device Interface adding or re-adding a cable after deleting - Cable.DoesNotExist

### ISSUE THREAD

**Issue by Yentel** [netbox-community/netbox#6869]
### NetBox version

v2.11.10 (upgraded today from 2.9.X

### Python version

3.8

### Steps to Reproduce

1. Navigate to the interfaces page of a certain device
2. Click on add cable and choose circuit termination on a given interface.
2. Select an available circuit and a side
3. Select the cable type (CAT6a in my case)
4. Create the cable

NOTE: I have been unable to reproduce this issue on a clone of our production netbox (same netbox version and python version).

NOTE: this also seems to occur when removing a cable between two device interfaces and trying to re-add the cable, so it does not seem to be limited to circuit terminations.

NOTE: If I choose another interface to re-create the cable, it works fine.

### Expected Behavior

A cable is created between the interface and the given circuit side or device interface

### Observed Behavior

 Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'dcim.models.cables.Cable.DoesNotExist'>

Cable matching query does not exist.

Python version: 3.8.10
NetBox version: 2.11.10

**Comment by TheNetworkGuy** [comment]
See #6747 and #6781 for more information.

TLDR; Caching issue. Set CACHE_TIMEOUT to 0.
